src/Flicking.ts

```diff
@@ -559,7 +566,11 @@ class Flicking extends Component<{
     const parsedElements = parseElement(element);
 
     const insertingIndex = Math.max(viewport.panelManager.getRange().min - parsedElements.length, 0);
-    return viewport.insert(insertingIndex, parsedElements);
+    const prependedPanels = viewport.insert(insertingIndex, parsedElements);
+
+    this.contentsReadyChecker.check(prependedPanels.map(panel => panel.getElement()));
```

Comment:
If `resizeOnContentsReady` is false, isn't `this.contentsReadyChecker` undefined? I think that You need to check everywhere you have used `this.contentsReadyChecker` . And the statement is repeated. How about making it a method?

Reply:
Thanks, I've fixed it.